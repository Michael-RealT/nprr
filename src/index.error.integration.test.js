"use strict";
jest.unmock("npm");
jest.mock("npm", () => ({
  load: jest.fn(callback => callback()),
  config: {
    sources: {
      project: {
        path: undefined
      }
    }
  }
}));

const main = require("./index.js");

it("Should throw an error if the project is not found", async () => {
  await expect(main()).rejects.toMatchSnapshot();
});
