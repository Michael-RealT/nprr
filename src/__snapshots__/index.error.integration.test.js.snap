// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should throw an error if the project is not found 1`] = `"No NPM project found"`;
